app/javascript/react/components/Reviews/ReviewTile.js

```diff
@@ -1,13 +1,24 @@
 import React from "react";
 import moment from "moment";
 
-const ReviewTile = ({ title, body, created_at }) => {
+const ReviewTile = ({ reviews }) => {
+
+  let reviewList
+  if (reviews) {
+    reviewList = reviews.map((review) => {
+      return (
+        <li key={review.id}>
+          <h3>{review.title}</h3>
+          <h5>Rating: {review.rating}</h5>
+          <p>{review.body}</p>
+          <p>Posted at: {moment(review.created_at).format("LL")}</p>
+        </li>
+      );
+    });
+  }
+  
   return (
-    <div>
-      <h3>{title}</h3>
-      <p>{body}</p>
-      <p>Posted at: {moment(created_at).format("LL")}</p>
-    </div>
+    <ul>{reviewList}</ul>
```

Comment:
I understand your implementation, but the naming and division of concerns is off. 

You have a ReviewTile component....that makes multiple "review tiles" (in this case li's). Its confusing.

Move the map logic into ReviewContainer. In your map, you should render a ReviewTile component
Reorganize the ReviewTile to be for making ONE review 
e.g. 
```
  <li key={review.id}>
          <h3>{review.title}</h3>
          <h5>Rating: {review.rating}</h5>
          <p>{review.body}</p>
          <p>Posted at: {moment(review.created_at).format("LL")}</p>
        </li>
```

Essentially: use the pattern we taught you in week 3 and 4, this breaks that pattern!